sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -1110,11 +1105,8 @@ export class MessageSession extends LinkEntity {
         this._receiver!.addCredit(maxMessageCount);
         let msg: string = "[%s] Setting the wait timer for %d seconds for receiver '%s'.";
         if (reuse) msg += " Receiver link already present, hence reusing it.";
-        log.batching(msg, this._context.namespace.connectionId, maxWaitTimeInSeconds, this.name);
-        totalWaitTimer = setTimeout(
-          actionAfterWaitTimeout,
-          (maxWaitTimeInSeconds as number) * 1000
-        );
+        log.batching(msg, this._context.namespace.connectionId, maxWaitTimeInMs / 1000, this.name);
```

Comment:
You know.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8169/commits/1313eb66765b5cc1fe2db2ce12c335d842f79db4